### DISCUSSION TITLE
Welcome to accelerated-computing-hub Discussions!

### DISCUSSION THREAD

**Discussion by nv-kriehl** [NVIDIA/accelerated-computing-hub#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tomhol** [comment]
Minor note for when absolute beginner goes thru the course (cuda-cpp):
In 01.02.04 - there are 3 steps (in the simulation, the main for loop) and there are also 3 items in the temperatures vector.
While this is completely unrelated, it may confuse some readers... For better clarity, we can use e.g. 7 steps in the simulation so it is crystal clear these two "3"s are not anyhow related :)
